js/src/wp-seo-admin.js

```diff
@@ -4,9 +4,7 @@
 /* jshint unused:false */
 
 /* jshint ignore:start */
-import React from "react";
-import ReactDom from "react-dom";
-import AlgoliaSearcher from "./kb-search/wp-seo-kb-search.js";
+import intialiseAlgoliaSearch from "./kb-search/wp-seo-kb-search-init";
```

Comment:
We use American English, which means `initialise` should be written as `initialize`. There is also a typo in the varname.